Dynamic_Bar.lua

```diff
@@ -100,28 +92,20 @@ function Type:Setup(icon)
  		values.triggerFunc = fnc
  	end
 
- 	icon:SetInfo("state;", 1)
```

Comment:
No longer needed if the state is being set in the update handler. However, not all code paths in `Value_OnUpdate` are doing this, so you may want to make some changes there to ensure all code paths are setting a state.